packages/api/src/command/medical/patient/get-patient-facility-matches.ts

```diff
@@ -169,36 +165,18 @@ function cwPatientToPatientMatch(patient: PatientNetworkLink): PatientFacilityMa
     country: addr.country ?? undefined,
   }));
 
-  const emails = patient.details.telecom?.reduce((acc: string[], curr) => {
-    if (curr.system === "email" && curr.value) {
-      acc.push(curr.value);
-    }
-    return acc;
-  }, []);
-
-  const phones = patient.details.telecom?.reduce((acc: string[], curr) => {
-    if (curr.system === "phone" && curr.value) {
-      acc.push(curr.value);
-    }
-    return acc;
-  }, []);
+  const emails = getCwPatientContactType(patient.details.telecom, "email");
+  const phones = getCwPatientContactType(patient.details.telecom, "phone");
 
   return {
-    firstName: toTitleCase(firstName),
+    firstName: firstName ? toTitleCase(firstName) : "",
     lastName: toTitleCase(lastName),
-    gender: patient.details.gender.display ? toTitleCase(patient.details.gender.display) : "UN",
+    gender: patient.details.gender.display
+      ? mapGenderAtBirthFromCw(patient.details.gender.code)
+      : "UN",
```

Comment:
Why checking for `display` and using `code`? Can't we just pass `code` and let the function return a fallback if not found?